subsys/bootloader/Kconfig

```diff
@@ -160,6 +160,7 @@ config PM_PARTITION_SIZE_PROVISION
 
 config PM_PARTITION_SIZE_B0_IMAGE
 	hex "Flash space reserved for B0_IMAGE"
+        default 0x8000 if !B0_MIN_PARTITION_SIZE
```

Comment:
Something weird with the indentation here?

Reply:
Yeah, fixed it.